### DISCUSSION TITLE
moving an object on the svg path in framer motion

### DISCUSSION THREAD

**Discussion by mordak-95** [motiondivision/motion#2323] [Q&A]
I want to move my own objects in an SVG on an path using the framer motion.
After a lot of searching, the only way I found on the internet was to use CSS, which created a conflict with the rest of my program.

**Comment by itaim18** [comment]
Been done ton of path animation lately, didn't quite understand  the moving an object part.
could you explain further maybe w/blockquote?

**Comment by Albx68** [comment]
https://pomax.github.io/bezierinfo/#yforx

you can look at this, I implemented something similar in react native using skia and reanimated, 

https://www.youtube.com/watch?v=xeLdmn3se1I

the concept should be slightly similar

hope it helps :D
